pkg/controller/installation/products/codeready/reconciler.go

```diff
@@ -496,28 +455,7 @@ func (r *Reconciler) reconcileKeycloakClient(ctx context.Context, serverClient p
 	return v1alpha1.PhaseCompleted, nil
 }
 
-func (r *Reconciler) getBackupConfig() resources.BackupConfig {
-	return resources.BackupConfig{
-		Namespace:     r.Config.GetNamespace(),
-		Name:          r.Config.GetNamespace(), //reusing namespace name because it already contains product name and is prefixed
-		BackendSecret: resources.BackupSecretLocation{Name: r.Config.GetBackendSecretName(), Namespace: r.ConfigManager.GetOperatorNamespace()},
-		Components: []resources.BackupComponent{
-			{
-				Name:     "codeready-postgres-backup",
-				Type:     "postgres",
-				Secret:   resources.BackupSecretLocation{Name: r.Config.GetPostgresBackupSecretName(), Namespace: r.Config.GetNamespace()},
-				Schedule: r.Config.GetBackupSchedule(),
-			},
-			{
-				Name:     "codeready-pv-backup",
```

Comment:
How do these backups actually work? If its backing up a pv does this need to be skipped when aws is used instead of in-cluster deployments?

Reply:
I think I need to reach out to the codeready team for more context here. 

But I am guessing that with every workspace that is created a PV is provisioned. So i reckon the backups are syncing between the PV and Postgres. 

I will reach out for clarification and let you know